components/switch/src/vwc-switch.scss

```diff
@@ -1,138 +1,165 @@
 @use 'sass:selector';
 @use '@vonage/vvd-design-tokens/build/scss/semantic-variables/scheme-variables';
 @use '@vonage/vvd-foundation/scss/mixins/connotation/config' with (
+//'Connotation.Success' is  *deprecated*  and will be removed in vivid-3. Use any of the other connotation predefined values:
 	$connotations-set: primary cta success alert,
 	$default: primary,
 );
 @use '@vonage/vvd-foundation/scss/mixins/connotation' as connotation;
 
+:host {
+	//css variables
+	--basic-nobe-size: 12px;
+	--translate: 16px;
+	--translate-natural: -16px;
+	--natural-50: #757575;
+	--surface: #f2f2f2;
+
+	contain: content;
+}
+
 .mdc-switch {
 	--mdc-theme-secondary: var(#{connotation.$vvd-color-connotation});
-
-	--vvd-switch-unchecked-background: var(#{scheme-variables.$vvd-color-neutral});
+	--vvd-switch-unchecked-background: trasparent;
 	--vvd-switch-unchecked-background-hover: var(#{scheme-variables.$vvd-color-neutral});
 	--vvd-switch-unchecked-background-focus: var(#{scheme-variables.$vvd-color-neutral});
-
 	--vvd-switch-disabled-background: var(#{scheme-variables.$vvd-color-neutral-30});
-	--vvd-switch-shadow-focus: 0, 0, 0, 0.5;
-
 	--mdc-theme-surface: var(#{scheme-variables.$vvd-color-neutral-10}); // unchecked thumb
 	--mdc-theme-on-surface: var(--vvd-switch-unchecked-background); // unchecked track
 
 	@include connotation.connotation;
-}
-
-:host {
-	border-radius: 16px;
-
-	.mdc-switch,
-	.mdc-switch__native-control {
-		width: 30px;
-		height: 20px;
-	}
-
-	.mdc-switch__track {
-		width: 100%;
-		height: 100%;
-		border-width: 0;
-		border-radius: 16px;
-	}
-
-	.mdc-switch__thumb-underlay {
-		top: 0;
-		left: 0;
-		width: 20px;
-		height: 100%;
 
+	&.mdc-switch--checked {
 		.mdc-switch__thumb {
-			width: 16px;
-			height: 16px;
-			border: 0 none;
+			--basic-nobe-size: 14px;
+			background-color: var(--surface);
 		}
-	}
-
-	.mdc-switch--checked {
 		.mdc-switch__thumb-underlay {
-			transform: translateX(10px);
+			transform: translateX(var(--translate));
```

Comment:
just a side note - use css variables with caution as they can lead to abuse if not handled correctly. I mean if there are states where they’re not defined by inner implementation, that’s a pit fall where piercing variables can damage the style. `translate` is a very generic variable name

Reply:
changed the variables names :)